_posts/2011-01-01-how-to-submit-apk.md

```diff
@@ -15,6 +15,35 @@ date: 2013-01-06 07:05:00
 }
 </style>
 
+<br>
+## Pre-Requisites
+
+Before uploading to Poynt developer portal, APKs must be digitally signed with android keys. Steps as follows:
```

Comment:
Let's rephrase:
_You need to sign your apk before uploading it in the developer portal:_